playbooks/install_stack.yaml

```diff
@@ -131,7 +154,7 @@
       tripleo_deploy_standalone: true
       tripleo_deploy_output_dir: "{{ ansible_env.HOME }}"
       tripleo_deploy_local_ip: "{{ public_api }}"
-      tripleo_deploy_control_virtual_ip: "{{ control_plane_ip }}"
+      tripleo_deploy_control_virtual_ip: "{{ public_api }}"
```

Comment:
this is something that I need to test against tripleo master because I'm not sure it works the same way between train & master for some reasons that I never understood honestly.